Parse/src/main/java/com/parse/Parse.java

```diff
@@ -608,7 +608,14 @@ private static void initializeParseHttpClientsWithParseNetworkInterceptors() {
     interceptors = null;
   }
 
-  // TODO(mengyan) Make public after we release the interceptor feature
+
+  /**
+   * Add a {@code ParseNetworkInterceptor}. You must invoke
+   * {@link #addParseNetworkInterceptor(ParseNetworkInterceptor)} before
```

Comment:
and this one should be `{@code addParseNetworkInterceptor(ParseNetworkInterceptor)}`